MethodSwitch/src/methodswitch/MethodSwitch.java

```diff
@@ -0,0 +1,170 @@
+
+package methodswitch;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author Seth Frosch
+ */
+public class MethodSwitch {
+
+    public static void main(String[] args) {
+        int number;
+        Scanner keyboard = new Scanner(System.in);
+        
+        // Reformatted the menu options to give one per line.
+        System.out.println("Enter '1' to play FIZZBUZZ.\n" + "Enter '2' to view the value of your chosen word.\n" + "Enter '3' to use the salary calculator.\n"
+                + "Enter '4' to check your grade.\n" + "Enter '5' to play Truth or Dare.\n" + "Enter '6' or any other number to EXIT.");
+        number = keyboard.nextInt();
+        
```

Comment:
If each menu item were on a separate line, it would make reading and editing it much easier.